.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -220,3 +249,90 @@ jobs:
             }
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
+
+
+  publish-docker-image:
+    timeout-minutes: 240
+    needs:
+      - start-publish-docker-image-runner-0
+      - publish-cdk
+    runs-on: runner-pool-${{ github.run_id }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          repository: airbytehq/airbyte
+          ref: master
+      - name: Install Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: Publish source-declarative-manifest
+        id: publish
+        env:
+          DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
+          DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          # Oracle expects this variable to be set. Although usually present, this is not set by default on Github virtual runners.
+          TZ: UTC
+        uses: Wandalen/wretry.action@master
+        with:
+          command: |
+            docker login -u ${DOCKER_HUB_USERNAME} -p ${DOCKER_HUB_PASSWORD}
+            ./tools/integrations/manage.sh publish airbyte-cdk/python false
+          attempt_limit: 3
+          attempt_delay: 5000 in # ms
+      - name: Prepare Sentry
+        uses: Wandalen/wretry.action@master
+        with:
+          attempt_limit: 3
+          attempt_delay: 5000 # in ms
+          command: curl -sL https://sentry.io/get-cli/ | bash || echo "sentry cli already installed"
+      - name: Set Name and Version Environment Vars
+        run: |
+          source tools/lib/lib.sh
+          echo "IMAGE_NAME=source-declarative-manifest" >> $GITHUB_ENV
+          echo "IMAGE_VERSION=$(_get_docker_image_version airbyte-cdk/python/Dockerfile)" >> $GITHUB_ENV
+      - name: Create Sentry Release
```

Comment:
This was part of the publish pipeline but I'm not sure how it works. I've asked for more information here https://airbytehq-team.slack.com/archives/C02UF50V9HA/p1677616768240659. In the meanwhile, I validated that the release is created in Sentry but could not validate how this release gets populated with information so this might be useless

Reply:
After some discussions with Pedro, this is not necessary as Sentry will be creating the release automatically if it does not exist